resources/assets/less/bem/quick-info.less

```diff
@@ -0,0 +1,60 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+
+.quick-info {
+	
+	// Elements
+	&__level {
+		font-style: normal;
+		color: #555;
+		font-family: @font-family-sans-serif;
+		font-size: 20px;
+		font-weight: 300;
+		margin: 0;
+		width: 100%;
+	}
+
+	&__roles {
+		flex: 1;
+	}
+
+	&__statistics {
+		display: flex;
+		flex-flow: column nowrap;
+		font-size: 12px;
+		font-weight: 300;
+		margin: 0;
+		padding: 0;
+		width: 100%;
+
+		> li {
+			line-height: 16px;
+			display: flex;
+			justify-content: space-between;
+
+			.text-right {
```

Comment:
`quick-info-statistics` is the block/wrapper element. `quick-info-statistics__statistics` is the bit at line 37.
But no, there isn't anything in `user-statistics`.